ui/admin/app/templates/scopes/scope/workers/worker/tags.hbs

```diff
@@ -61,11 +62,80 @@
                 <Hds::Badge @text={{B.data.type}} />
               {{/if}}
             </B.Td>
+            <B.Td>
+              {{#if (eq B.data.type this.tagTypeApi)}}
+                <Hds::Dropdown as |dd|>
+                  <dd.ToggleIcon
+                    @icon='more-horizontal'
+                    @text='Overflow Options'
+                    @hasChevron={{false}}
+                    @size='small'
+                  />
+                  {{#if (can 'removeTags worker' @model)}}
+                    <dd.Interactive
+                      @text={{t 'resources.worker.tags.actions.remove'}}
+                      {{on 'click' (fn this.toggleModal 'remove' B.data)}}
+                    />
+                  {{/if}}
+                </Hds::Dropdown>
+              {{/if}}
+            </B.Td>
           </B.Tr>
         </:body>
       </Hds::Table>
     {{/if}}
 
+    {{#if this.removeModal}}
+      <Hds::Modal
+        class='remove-tag-modal'
+        @color='critical'
+        @onClose={{fn this.toggleModal 'remove'}}
+        as |M|
+      >
+        <M.Header @icon='alert-diamond'>
+          {{t 'resources.worker.tags.remove.title'}}
+        </M.Header>
+        <M.Body>
+          <Hds::Text::Body @color='primary'>
+            {{this.tagDisplayName}}
+            {{t 'resources.worker.tags.remove.warning'}}
+          </Hds::Text::Body>
+          {{! template-lint-disable no-autofocus-attribute }}
+          <Hds::Form::TextInput::Field
+            name='removal-confirmation'
+            autofocus={{true}}
```

Comment:
What is the purpose of the autofocus here? as per your video in the PR description, once the dialog opens, the field is NOT autofocusing (you have to click on it).

Reply:
Great question. I forgot to update the video. I was looking at some [HDS documentation for modals](https://helios.hashicorp.design/components/modal?tab=code#form-within-a-modal-dialog) and they recommend adding the `autofocus` attribute if there is a form in the modal. So I added it after the original video was made. Sorry for the confusion.

Reply:
Video updated!

Reply:
Awesome, now makes sense 😃  Thank you @cameronperera